public/content/developers/docs/apis/javascript/index.md

```diff
@@ -31,7 +31,7 @@ Using providers, these libraries allow you to connect to Ethereum and read its d
 ```js
 // A Web3Provider wraps a standard Web3 provider, which is
```

Comment:
Thanks @pglekshmi! Seeing support for this in the [v6 ethers docs](https://docs.ethers.org/v6/getting-started/)... Let's also update the comment so it aligns:

```suggestion
// A BrowserProvider wraps a standard Web3 provider, which is
```